### PR TITLE
Plots

### PR DESCRIPTION
**Pull request by sjanssen2**
This PR shall add in 
a) options to activate outside grammar generation and b) let user decide about level of detail when generating grammar plots
image

Furthermore, the plot as image is now presented as a third tab:
image

### CODE REVIEW

**Comment by kmaibach** [comment]
I like that there is now a tab for the picture.
One thing I noticed in the current online version is: The picture is quite small in its height.
Is the online version the same as this one? If so, could you somehow rescale the plot? 

And we need some documentation.  Or is there a place already where the levels of visualization are explained?

Also, I noticed two other errors regarding some other features of the website, which I will open as issues.

**Comment by sjanssen2** [comment]
Scaling is in fact an issue. I do not yet have a good solution as some graphs can be really huge: "genefinder" is extremely wide, while others can be quite long :-/

@Documentation: you are absolutely right. We need two things a) some type of legend to explain the visual aspects of the plot and b) an extensive text explaining the meaning of the annotations. This will only be understandable if we at the same time explain fundamental concepts. A larger project.

File: templates/bellman.html [hunk 1]
```diff
@@ -550,6 +571,7 @@ <h1>Welcome to Bellman's Cafe</h1>
         <ul class="tabs-list">
             <li ><a href="#tab1">Input</a></li>
             <li ><a href="#tab2">Gapc</a></li>
+            <li ><a href="#tab_grammar">grammar plot</a></li>
```
Comment (Anna-Rehm) [inline templates/bellman.html:574]: Perhaps "grammar plot" should have capitalized letters just like the other tabs for a more coherent look -> Grammar Plot

Comment (sjanssen2) [reply templates/bellman.html]: good point. Fixed

**Pull request merged**